  MODPOST 932 modules
ERROR: "__bad_udelay" [drivers/net/wireless/ath5k/ath5k.ko] undefined!

--- a/drivers/net/wireless/Kconfig	2008-01-19 19:13:01.000000000 +0000
+++ b/drivers/net/wireless/Kconfig	2008-01-19 19:13:25.000000000 +0000
@@ -658,7 +658,7 @@
 
 config ATH5K
 	tristate "Atheros 5xxx wireless cards support"
-	depends on PCI && MAC80211 && WLAN_80211 && EXPERIMENTAL
+	depends on PCI && MAC80211 && WLAN_80211 && EXPERIMENTAL && (!ARM || BROKEN)
 	default m
 	---help---
 	  This module adds support for wireless adapters based on
